Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
 RUN gem install \
         fluentd:1.12.1 \
         concurrent-ruby:1.1.8 \
-        google-protobuf:3.15.6 \
+        google-protobuf:3.9.2 \
```

Comment:
Why specifically this version?

In this file's history I can see that there's been a couple of bumps of this package already but only on the `3.15.x` minor.

Reply:
I did not verify when exactly the problem appeared. We know the image `1.12.0-sumo-1` worked fine and it had the version `3.9.2` of google-protobuf. Image `1.12.0-sumo-2` does not work fine and it has version `3.15.6`.

Reply:
Please block dependabot for this package

Reply:
as well as rakefiles

Reply:
@sumo-drosiek Done in https://github.com/SumoLogic/sumologic-kubernetes-fluentd/pull/160/commits/5029b2979288e9c1b7b9627ee67335d66842b8b7.

Reply:
Also updated gemspec in https://github.com/SumoLogic/sumologic-kubernetes-fluentd/pull/160/commits/8077ed68ced406996a31e262b42f8473e1dac248.